src/util/unicode.cpp

```diff
@@ -256,8 +256,14 @@ const char **narrow_argv(int argc, const wchar_t **argv_wide)
   return argv_narrow;
 }
 
-std::wstring utf8_to_utf16(const std::string& in)
+std::wstring utf8_to_utf16be(const std::string& in)
 {
   std::wstring_convert<std::codecvt_utf8_utf16<wchar_t> > converter;
   return converter.from_bytes(in);
 }
+
+std::wstring utf8_to_utf16le(const std::string& in)
+{
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t,0x10ffffUL,std::codecvt_mode::little_endian> > converter;
```

Comment:
Is there some macro or named constant that can be used in place of the hex value? Else it would be nice to have some comment explaining why it's exactly this bit pattern.

Also, the line should be broken somewhere before the 80th (or even 70th) character.